tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -80,7 +80,6 @@ then
   for RUBY_VERSION in 2.5.0 2.7.0 3.0.0; do
     echo "Installing ruby-${RUBY_VERSION}"
     time rvm install "ruby-${RUBY_VERSION}"
-    time gem install bundler -v 1.17.3 --no-document
```

Comment:
would it make sense to pin bundler version to some specific version?

Reply:
The only problem with that is I think it could lead to the same situation of what seems to have happened here again, where the rubygems version could become incompatible with the bundler version.